openvdb_houdini/CMakeLists.txt

```diff
@@ -223,7 +223,7 @@ TARGET_LINK_LIBRARIES ( openvdb_houdini
 
 IF ( OPENVDB_ENABLE_RPATH )
   # @todo There is probably a better way to do this for imported targets
-  SET ( RPATHS )
+  SET ( RPATHS "" )
```

Comment:
Out of interest what was happening here? set without an arg is basically an unset

Reply:
This was the error:
```
CMake Error at openvdb_houdini/CMakeLists.txt:206 (LIST):
  list sub-command REMOVE_DUPLICATES requires list to be present.
```

Line number has since changed, it was this line I believe:

`  LIST ( REMOVE_DUPLICATES RPATHS )`

Reply:
Interesting, not seeing it but looks good none the less